app/assets/stylesheets/application.scss

```diff
@@ -573,3 +573,7 @@ table {
 .masked-input {
   font-family: 'AllBullets';
 }
+
+.code-in-table {
```

Comment:
There should already be styling of `<code>` tags from Bootstrap, is this needed? (and should it be `table code`?)